pediatric_rootlets/crop_composed_images.py

```diff
@@ -0,0 +1,105 @@
+"""
+This script crops the T2w image (label files (SC seg, PMJ, ...)) based on a specific intervertebral disc
+(specified by "-x").
+
+The script requires the SCT conda environment to be activated (because we import the SCT's Image class):
+    source ${SCT_DIR}/python/etc/profile.d/conda.sh
+    conda activate venv_sct
+"""
+
+import os
+from spinalcordtoolbox.image import Image, zeros_like
+from argparse import RawTextHelpFormatter
+import argparse
+import numpy as np
+
+
+def get_parser():
+    """
+    parser function
+    """
+
+    parser = argparse.ArgumentParser(
+        description='The script crops the T2w image (label files (SC seg, PMJ, ...)) based on a specific intervertebral disc (specified by "-x").',
+        formatter_class=RawTextHelpFormatter,
+        prog=os.path.basename(__file__)
+    )
+    parser.add_argument(
+        '-i_composed',
```

Comment:
A common convention is to use `-` instead of `_`

```suggestion
        '-i-composed',
```

Reply:
Fixed in [9f0c979](https://github.com/ivadomed/model-spinal-rootlets/pull/53/commits/9f0c9792791d6ac3e248bd86e1f59031fa6e4d47).